packages/salesforcedx-vscode-core/src/settings/pushOrDeployOnSave.ts

```diff
@@ -45,6 +47,7 @@ export class DeployQueue {
   }
 
   public async enqueue(document: vscode.Uri) {
+    telemetryService.sendEventData('deployOnSaveEnqueue');
```

Comment:
This event will send every enqueue - going back and forth on whether we need this or not.

Reply:
Having the metric in line 93 already contains the number of items to be deployed so I think we could remove this one.